node-runner-cli/Makefile

```diff
@@ -3,11 +3,10 @@ install:
 	$(shell chmod +x add-version.sh)
 	$(shell ./add-version.sh)
 	pip install pipenv
-	pipenv lock -r > requirements.txt
 
 .PHONY: output
 output-ubuntu-focal: install
-	DOCKER_BUILDKIT=1 docker build --output type=local,dest=out/ubuntu/focal --progress plain -f Dockerfile.ubuntufocal  .
+	DOCKER_BUILDKIT=1 docker build --platform linux/amd64 --output type=local,dest=out/ubuntu/focal --progress plain -f Dockerfile.ubuntufocal  .
```

Comment:
This can be a variable like we have here https://github.com/radixdlt/radixdlt/blob/main/radixdlt-core/Makefile#L4